src/main/java/codeu/model/data/Activity.java

```diff
@@ -24,15 +26,15 @@
     private final UUID id;
```

Comment:
How come you are using a final class?

Reply:
I realize now that this is part of the code from before the start date. And you might not know why specifically.

Reply:
Ya it was set as final because we currently do not perform edits to models. In that case we will remove the `final` modifier from the properties